src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -274,29 +281,29 @@ class ClusterMonitorActor(val cluster: Cluster,
         case Unknown | Creating | Updating =>
           Future.successful(NotReadyCluster(googleStatus, googleInstances))
         // Take care we don't restart a Deleting or Stopping cluster if google hasn't updated their status yet
-        case Running if clusterStatus != Deleting && clusterStatus != Stopping && clusterStatus != Starting && runningInstanceCount == googleInstances.size =>
+        case Running if leoClusterStatus != Deleting && leoClusterStatus != Stopping && leoClusterStatus != Starting && runningInstanceCount == googleInstances.size =>
```

Comment:
Just renamed var to `leoClusterStatus` for clarity.

Reply:
Why was `clusterStatus` not clear?

Reply:
Because there's a `googleStatus` which comes from Google and a `leoClusterStatus` which comes from the Leo DB (both are of type `ClusterStatus`).

Reply:
I suspected so and then couldn't see a dedicated type from a quick look. Not for this PR but it'd be clearer to me to have a separate type (or subtype) for Google statuses.